signale.js

```diff
@@ -257,6 +258,18 @@ class Signale {
       return {label, span};
     }
   }
+
+  update(type, ...args) {
+    const streams = type.stream || this._formatStream(this._stream);
```

Comment:
`this._formatStream(type.stream || this._stream);`

As type.stream can also be an array or a single stream.